std/base64.d

```diff
@@ -24,7 +24,6 @@
  * ubyte[] decoded = Base64.decode("FPucA9l+");
  * assert(decoded == [0x14, 0xfb, 0x9c, 0x03, 0xd9, 0x7e]);
  * -----
- *
```

Comment:
> This doesn't seem to have any effect.

Scratch that. It doesn't have an effect on the DDOX page, but it has an effect on the Ddoc page.